params/neils_bird.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 3.1415 
+    
+# e.g. "anas_krystallinus"    
+species.name <- "Coco_thekiller"
```

Comment:
We've just had Coco thekiller added, could you choose a different bird name please.

Reply:
Thank you for your comments.